pkgs/applications/misc/fusee-interface-tk/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv , fetchFromGitHub , python3 , makeWrapper }: 
+
+let pythonEnv = python3.withPackages(ps: [ ps.tkinter ps.pyusb ]); 
+in stdenv.mkDerivation { 
+
+  pname = "fusee-interface-tk";
+  version = "1.0.0";
+  
+  src = fetchFromGitHub { 
+    owner = "nh-server";
+    repo = "fusee-interfacee-tk";
+    rev = "3053a5ffef8efb5ae3da9666f8f53285c5a904b2";
```

Comment:
```suggestion
    rev = "V${version}";
```

Reply:
I can't seem to get this working on my end.